fastlane/lanes/codesign.rb

```diff
@@ -19,22 +19,25 @@
       puts "\t#{identifier}"
     end
 
+    team_id = get_required_env('EXT_EXPORT_TEAM_ID')
```

Comment:
This little DRY refactor has nothing to do with the API key. But, it caught my eye and was fast to execute.